roles/gateway/tasks/main.yml

```diff
@@ -19,6 +19,7 @@
 # Manage OpenVPN users list using secrets repo
 - include: users.yml
   tags:
+    - always
```

Comment:
as to "why not": what if I want to set up an openvpn server but not set up the users file?